doc/Administration-guide/System-installation/Cluster-setup/index.md

```diff
@@ -1,6 +1,6 @@
----
+﻿---
```

Comment:
There is a non-printable symbol at the beginning of this file. It causes the problem described in [KAA-1668](http://jira.kaaproject.org/browse/KAA-1668)